### ISSUE TITLE
Linux_android spell_check_test is 28.00% flaky

### ISSUE THREAD

**Issue by fluttergithubbot** [flutter/flutter#113710]
The post-submit test builder `Linux_android spell_check_test`, which has been marked `bringup: true`, had 28 flakes over past 100 commits.

One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1806
Commit: https://github.com/flutter/flutter/commit/3f89d6393f249a844a5ac20cb122526f13131cc3

Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1806
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1802
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1801
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1799
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1795
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1794
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1793
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1791
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1790
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1788
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1786
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1778
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1775
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1774
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1770
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1769
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1767
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1764
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1761
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1759
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1757
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1741
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1730
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1728
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1727
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1726
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1723
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1719
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1718
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/1712

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test

Please follow https://github.com/flutter/flutter/wiki/Reducing-Test-Flakiness#fixing-flaky-tests to fix the flakiness and enable the test back after validating the fix (internal dashboard to validate: go/flutter_test_flakiness).

**Comment by keyonghan** [comment]
https://github.com/flutter/cocoon/pull/2236 should avoid such bugs being created.

**Comment by jmagman** [comment]
Going to reopen this one, I don't think the flake is being tracked anywhere.

**Comment by jmagman** [comment]
https://ci.chromium.org/p/flutter/builders/staging/Linux_android%20spell_check_test

**Comment by jmagman** [comment]
@camsim99 can you take a look?

**Comment by camsim99** [comment]
This test is going to become a manual test since the spell checker does not have deterministic behavior, just waiting on a fix for the iOS part to land. Then, I'll submit https://github.com/flutter/flutter/pull/116541.

**Comment by jmagman** [comment]
> become a manual test 

Hmm, who is going to manually run it and validate it doesn't regress?

Does the iOS one have deterministic behavior?  It looks like it's just misconfigured with `deviceOperatingSystem = DeviceOperatingSystem.android;`.

**Comment by camsim99** [comment]
> > become a manual test
> 
> Hmm, who is going to manually run it and validate it doesn't regress?
> 
> Does the iOS one have deterministic behavior? It looks like it's just misconfigured with `deviceOperatingSystem = DeviceOperatingSystem.android;`.
> 
> https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8794889598187185393/+/u/run_spell_check_test/test_stdout

I'm not sure yet if the iOS side is flaky. I created a separate task to test the fix I just landed: https://github.com/flutter/flutter/pull/116222. I can land it first before making the test manual to see how it behaves.

There is full test coverage of spell check behavior on the engine and framework side without this integration test. The only advantage to having the integration test is making sure that the native spell checker is doing its job. That has, unfortunately, proven itself to not be consistent (at least on Android). Because the integration test was always just a "nice-to-have" and this test is continuously flaky, I think we need to make it manual. I can definitely add comments in relevant files to mention the test and how it can be used to validate behavior.

**Comment by fluttergithubbot** [comment]
[staging pool] current flaky ratio for the past (up to) 100 commits is 41.00%. Flaky number: 41; total number: 100.
One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3392
Commit: https://github.com/flutter/flutter/commit/6819f72a91553282103362cf65cd1d6c97aa479a
Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3392
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3391
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3390
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3388
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3387
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3386
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3385
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3384
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3380
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3379
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3377
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3373
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3371
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3370
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3369
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3367
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3363
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3362
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3361
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3360
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3359
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3358
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3357
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3356
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3355
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3349
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3344
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3337
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3325
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3324
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3323
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3318
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3314
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3297
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3280
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3279
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3278
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3274
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3273
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3269
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3268
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3262
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3256
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3255
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3254
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3252
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3249
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3245
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3244
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3242
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3239
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3233
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3228

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test

**Comment by camsim99** [comment]
https://github.com/flutter/flutter/issues/117498 This issue to track investigating best way to handle this.

**Comment by camsim99** [comment]
Update: Increased the timeout duration in the hopes of seeing the root cause of the flakiness better. Working on https://github.com/flutter/flutter/issues/117498 by investigating solutions for the devices learning words.

**Comment by fluttergithubbot** [comment]
[staging pool] current flaky ratio for the past (up to) 100 commits is 38.00%. Flaky number: 38; total number: 100.
One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3738
Commit: https://github.com/flutter/flutter/commit/a40fb9bdd76a5d8a3f135feaedc3b41b8996d482
Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3738
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3737
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3727
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3723
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3714
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3713
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3712
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3711
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3710
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3709
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3708
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3707
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3704
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3703
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3702
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3701
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3681
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3680
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3678
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3677
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3676
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3675
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3672
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3652
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3651
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3644
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3643
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3638
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3637
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3636
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3632
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3631
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3630
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3628
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3627
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3619
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3614
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3612
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3606
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3605
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3604
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3602
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3600
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3599
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3595
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3594
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3587
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3583
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3581
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3576

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test

**Comment by camsim99** [comment]
Update: I've been able to recreate this locally, so I have confirmed that either the native Android spell checker learns word or Gboard learns words. However, I have not yet found a fix. I know someone who may have written a similar test or have more context on what's happening, so I will follow up with them. Depending on the result of that, we will add a fix for this test or make it a manual test.

**Comment by fluttergithubbot** [comment]
[staging pool] current flaky ratio for the past (up to) 100 commits is 41.00%. Flaky number: 41; total number: 100.
One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3871,%203869,%203870
Commit: https://github.com/flutter/flutter/commit/0eaa83ad63d04b658dd2d7aaa9a718ebb0925312
Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3871
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3870
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3869
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3868
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3866
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3853
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3852
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3845
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3844
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3831
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3824
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3820
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3816
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3815
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3808
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3807
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3803
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3798
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3794
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3792
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3784
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3783
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3781
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3779
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3778
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3777
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3776
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3773
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3769
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3762
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3761
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3760
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3759
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3756
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3749
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3742
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3738
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3737
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3727
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3723
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3714
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3713
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3712
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3711
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3710
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3709
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3708
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3707
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3704
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3703
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3702
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3701

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test

**Comment by camsim99** [comment]
The fix for this flakiness is turning off the option for Gboard to personalize spell check results. Investigating how to turn this option off for devices running this test in the devicelab. If this is not possible, we will investigate ways to write this integration test with misspelled text that is generated differently for each test run.

**Comment by fluttergithubbot** [comment]
[staging pool] current flaky ratio for the past (up to) 100 commits is 27.00%. Flaky number: 27; total number: 100.
One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4104,%204103
Commit: https://github.com/flutter/flutter/commit/318f8758ba9d76d9884bafa511eacfbaace73c10
Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4104
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4103
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4099
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4098
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4097
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4095
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4091
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4090
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4089
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4070
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4060
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4049
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4046
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4045
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4040
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4035
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4030
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4026
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4025
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4019
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4018
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4017
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4015
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4004
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4003
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3999
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3998
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3997
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3987
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3982
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3981
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3980
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3977
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3973
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3970
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3962
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3955
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3954
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3953
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3941
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3940
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/3934

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test

**Comment by camsim99** [comment]
After following up with folks who work on Gboard and @CaseyHillers, it has come to my attention that it is not possible to turn off the Gboard personalization setting for a devicelab test via the command line, but a workaround is to implement a custom IME and use it in the test. Filed https://github.com/flutter/flutter/issues/119533 to track the progress on using this workaround.

**Comment by christopherfujino** [comment]
@camsim99 shall we lower the priority on this issue as it is not blocking the tree?

**Comment by fluttergithubbot** [comment]
[staging pool] current flaky ratio for the past (up to) 100 commits is 25.00%. Flaky number: 25; total number: 100.
One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4303
Commit: https://github.com/flutter/flutter/commit/a752c2f1545a0bb663cfeda55e236ce4a3fe2474
Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4303
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4297
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4282
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4278
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4269
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4268
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4267
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4258
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4245
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4236
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4226
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4225
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4224
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4223
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4213
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4212
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4211
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4210
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4209
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4208
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4207
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4206
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4205
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4204
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4203
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4202
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4192
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4191
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4190
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4186
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4177
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4169
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4162
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4158
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4157
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4156
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4153

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test

**Comment by camsim99** [comment]
Created https://github.com/flutter/flutter/pull/120144 to fix this issue for now by deleting the test for Android. Will revisit when I have the bandwidth: https://github.com/flutter/flutter/issues/119533

**Comment by fluttergithubbot** [comment]
[staging pool] current flaky ratio for the past (up to) 100 commits is 15.00%. Flaky number: 15; total number: 100.
One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4512
Commit: https://github.com/flutter/flutter/commit/1e6e6d41e3f747640119cee8e58914547ec44377
Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4512
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4511
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4508
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4505
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4504
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4500
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4488
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4478
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4477
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4475
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4468
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4430
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4422
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4389
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4370
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4355
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4352
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_android%20spell_check_test/4351

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux_android%20spell_check_test
